### PR TITLE
add source to metadata

### LINKED ISSUE #34: /model/metadata does not identify model source
```
{
  "id": "inception_resnet_v2-keras-imagenet",
  "name": "inception_resnet_v2 Keras Model",
  "description": "inception_resnet_v2 Keras model trained on ImageNet",
  "type": "image_classification",
  "source": null,
  "license": "Apache v2"
}
```

### PR DESCRIPTION
**Pull request by bdwyer2**
resolves #34

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -45,5 +45,6 @@
     'name': '{} Keras Model'.format(MODEL_NAME),
     'description': '{} Keras model trained on ImageNet'.format(MODEL_NAME),
     'type': 'image_classification',
-    'license': '{}'.format(MODEL_LICENSE)
+    'license': '{}'.format(MODEL_LICENSE),
```
Comment (xuhdev) [inline config.py]: (Not strictly relevant to this PR) Do we need a `format` here? Can we simply do
```suggestion
    'license': MODEL_LICENSE,
```

**Comment by MLnick** [comment]
@bdwyer2 we should add a test case also in 

https://github.com/IBM/MAX-Inception-ResNet-v2/blob/master/tests/test.py#L23

**Comment by ptitzler** [comment]
That should have been part of the PR ):

**Comment by MLnick** [comment]
Yes I was too hasty in merging - can do a follow up PR with the tests

**Pull request merged**